### PR TITLE
Computerome

### PR DESCRIPTION
**Pull request by marcmtk**
Adding an institutional profile for [Computerome](https://www.computerome.dk/) a Danish supercomputer for life sciences. 

I've followed the guide from [Bytesize 10](https://nf-co.re/events/2021/bytesize-10-institutional-profiles) and have tested using nf-core/viralrecon with either of `-c` or `-profile computerome --custom_config_base` syntax.

As computerome requires PBS -A and -W tags to be set i've followed the example of the uppmax configuration file and introduced a params.project. Jobs will fail if it is not set, but they only fail upon first submitting a job to the scheduler. It would be nice if it could fail faster, but i don't know how that would be done.

### CODE REVIEW

File: conf/computerome.config [hunk 1]
```diff
@@ -0,0 +1,29 @@
+//Profile config names for nf-core/configs
+params {
+  config_profile_description = 'Computerome 2.0 cluster profile provided by nf-core/configs.'
+  config_profile_contact = 'Marc Trunjer Kusk Nielsen (@marcmtk)'
+  config_profile_url = 'https://www.computerome.dk/'
+  project = null
+  schema_ignore_params = "project,genomes,cache_dir,modules"
```
Comment (drpatelh) [inline conf/computerome.config:7]: `genomes` and `modules` should be ignored in the pipeline `nextflow.config` and so don't need to be set here?

Comment (drpatelh) [reply conf/computerome.config:7]: Like what you have done with `cache_dir` and `project` though 👌🏽

File: conf/computerome.config [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  
+  //Thin nodes with 192GB and Fat nodes with ~1500GB. Torque should be allowed to handle this
+  max_memory = 1500.GB 
+  max_cpus = 40
+  
+  //There is no max walltime on the cluster, but a week seems sensible if not directly specified
+  max_time = 168.h 
+}
+params.cache_dir = "/home/projects/$params.project/scratch"
```
Comment (drpatelh) [inline conf/computerome.config:16]: Specify this in the `params` scope above?

**Review by drpatelh — COMMENTED** [review]
Looks great overall @marcmtk ! Just a few comments to address. I sent you an invite to become a member of the org too :)

**Review by drpatelh — APPROVED** [review]
LGTM! Feel to merge after any tests you need to do.

**Review by marcmtk — COMMENTED** [review]
Sorry, hadn't seen you'd already approved the changes and cleaned it a little more :)

**Pull request merged**